katello/katello-backup

```diff
@@ -7,14 +7,20 @@ require "yaml"
 
 @options = {}
 @dir = nil
+@is_foreman_proxy_content = !`rpm -qa | grep foreman-proxy-content`.empty?
+@databases = ['pulp', 'postgres', 'mongodb']
+@options[:features] = []
```

Comment:
We shouldn't be referring to an option feature if the flag was not set. If you need to check for specific features, I would do something along the lines of 
`features ||= @options[:features] || []`